code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -15,9 +15,10 @@
 
 /datum/symptom/heal/Start(datum/disease/advance/A)
 	if(!..())
-		return
+		return FALSE
 	if(A.properties["stage_rate"] >= 6) //stronger healing

```

Comment:
Shouldn't this make a threshhold for every  subclass? I don't think that's intentional.

Reply:
superficial healing was ported from an ancient /tg/ pr that never went through cuz /tg/ is gay. most of the code here is just copied from that.